Tests/LibWeb/AccessibilityBridgeMac/harness/__init__.py

```
@@ -0,0 +1,37 @@
+    "find_all_by_role",
+    "find_first_by_role",
+    "get_attribute_names",
+    "get_parameterized_attribute_names",
+    "get_parameterized_attribute_value",
+    "ladybird_for_fixture",
+    "role_path",
+    "supports_attribute",
+    "wait_for",
+    "wait_for_descendant_by_role",
+    "walk",
+]
```

Ruff (0.15.21)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)